tests/functional/sync_v2_sign_up.js

```diff
@@ -66,20 +66,9 @@ define([
         .findByCssSelector('#fxa-choose-what-to-sync-header')
         .end()
 
-        // test that autofocus attribute has been applied sanely
-        .findByTagName('input')
+        // test that autofocus attribute has been applied to submit button
+        .findById('submit-btn')
           .getAttribute('autofocus')
```

Comment:
This doesn't actually test anything. If I remove the `autofocus` attribute from the element, the test does not fail.